### PR TITLE
Jpuerto/dc helper dag

### PR DESCRIPTION
**Pull request by jpuerto-psc**
This PR adds a helper DAG for @bhonick to be able to bulk update status, ingest_task, assigned_to_group_name, and description for uploads and datasets.

### CODE REVIEW

File: src/ingest-pipeline/airflow/dags/bulk_update_entities.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        for uuid in uuids:
+            # If this fails out then we know its something other than an upload or dataset
+            try:
+                pythonop_get_dataset_state(dataset_uuid_callable=lambda **kwargs: uuid, **kwargs)
+                filtered_uuids.append(uuid)
+            except Exception as e:
+                print(f"{uuid} is neither a dataset nor an upload and will be skipped.")
+                print(repr(e))
+
+        kwargs["ti"].xcom_push(key="uuids", value=filtered_uuids)
```
Comment (sunset666) [inline src/ingest-pipeline/airflow/dags/bulk_update_entities.py:79]: There is probably no point in storing the uuids in the Airflow DB if they are not going to be used downstream.

File: src/ingest-pipeline/airflow/dags/bulk_update_entities.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+        for uuid in uuids:
+            endpoint = f"entities/{uuid}"
+            try:
+                response = http_hook.run(endpoint, headers=headers, data=json.dumps(metadata))
+                print("response: ")
+                pprint(response.json())
+                time.sleep(10)
+            except:
+                print(f"ERROR: UUID {uuid} could not be updated.")
```
Comment (sunset666) [inline src/ingest-pipeline/airflow/dags/bulk_update_entities.py:114]: Might as well have a sleep in the exception?

**Review by sunset666 — CHANGES_REQUESTED** [review]

File: src/ingest-pipeline/airflow/dags/bulk_update_entities.py (hunk 1 above)
Comment (jpuerto-psc) [reply src/ingest-pipeline/airflow/dags/bulk_update_entities.py]: Comments addressed

File: src/ingest-pipeline/airflow/dags/bulk_update_entities.py (hunk 2 above)
Comment (jpuerto-psc) [reply src/ingest-pipeline/airflow/dags/bulk_update_entities.py:111]: Moved sleep to outside the try/except block

**Review by sunset666 — APPROVED** [review]

**Pull request merged**